pkgs/applications/virtualization/docker/default.nix

```diff
@@ -233,16 +228,16 @@ rec {
       installShellCompletion --zsh  ./contrib/completion/zsh/_docker
     '' + lib.optionalString (stdenv.hostPlatform == stdenv.buildPlatform) ''
       # Generate man pages from cobra commands
-      echo "Generate man pages from cobra"
-      mkdir -p ./man/man1
-      go build -o ./gen-manpages github.com/docker/cli/man
-      ./gen-manpages --root . --target ./man/man1
+      #echo "Generate man pages from cobra"
+      #mkdir -p ./man/man1
+      #go build -o ./gen-manpages github.com/docker/cli/man
+      #./gen-manpages --root . --target ./man/man1
     '' + ''
       # Generate legacy pages from markdown
-      echo "Generate legacy manpages"
-      ./man/md2man-all.sh -q
+      #echo "Generate legacy manpages"
+      #./man/md2man-all.sh -q
 
-      installManPage man/*/*.[1-9]
+      #installManPage man/*/*.[1-9]
```

Comment:
Should we execute this conditionally instead of removing it also from older versions ?
As it is today, it's kind-of a breaking changes (removes the man pages from older docker versions), but we could try to make it so only `docker_24` not having it generated ?

Reply:
Good point, will adjust this.